pkg/types/baremetal/validation/platform.go

```diff
@@ -85,5 +85,25 @@ func ValidatePlatform(p *baremetal.Platform, n *types.Networking, fldPath *field
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("bootstrapHostIP"), p.BootstrapProvisioningIP, err.Error()))
 	}
 
+	if p.ProvisioningInterface == "" {
+		allErrs = append(allErrs, field.Invalid(fldPath.Child("provisioningInterface"), p.ProvisioningInterface, "Baremetal provisioning interface unset."))
+	}
+
+	if p.ProvisioningNetworkCIDR == "" {
+		allErrs = append(allErrs, field.Invalid(fldPath.Child("provisioningNetworkCIDR"), p.ProvisioningNetworkCIDR, "Baremetal provisioning network CIDR not set."))
+	}
+
+	if p.ProvisioningIP == "" {
```

Comment:
If this approach is acceptable we'll need to add actual validation for all of these similar to what I did in https://github.com/openshift/installer/pull/2320

Reply:
Yes just a POC.  The tricky part is that a lot of these values don't work with the validations.. eg the provisioning IP is in the form of a CIDR, the interface doesn't exist so it errors out etc.